pages/Institution.xml

```diff
@@ -28,11 +28,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
          <div type="level2">
             <head>Specific encoding for repositories</head>
-            <p>For repositories where manuscripts are preserved all general guidelines and especially those for <ref target="places">places</ref> are valid.
+            <p>All collections of manuscripts (churches, monasteries, archives, libraries, museums, private collections, etc) need a record of INS-type, stored in the Institutions repository.  A repository is a different conceptual entity from Place: potentially a repository/library may move in space; and at the same time multiple repositories may share the same coordinates. (The fact that churches in Ethiopia are also repositories may generate some confusion in this regard as their place is very unlikely to change; please check the <ref target="placeOrInstitution">Place or Repository</ref> Guidelines section to overcome the confusion and decide if you need a repository record.</p>
```

Comment:
The beginning is a bit complicated and will not be easy for a beginner. I would suggest: 
"All collections of manuscripts (kept in ...., private hands) need a record of INS-type, stored under Institutions.
 We consider each of those institutions a repository of manuscripts, a conceptual entity different from that of Place:... " I tried to call "Institution repository" in any different way, to disambiguate from what follows.
I would drop a small passage at "may generate some confusion; please check..." (churches sometimes moves, though not too frequently, but for this one only needs to take out tabot; cp. Dabra Ma'eso collection which in 2010 had a number of books moved there from a different repository which could not be located). In the course of the text, I would just repeat from time to time "repository of manuscripts", as "repository" alone is a little undifferentiated and has further modern connotations that may be confusing.

Reply:
Your example about Dabra Ma'eso is wrong, it is not about the change of location of a repository but about the change of mss from one repository to another. 
What is meant under repository's change of place is
e.g. Private Collection Denis Nosnitsin had address in Hamburg and now in Norderstedt, but it is still the same collection.
e.g. Library of HLZ was in the main building and now in AT, and then will be in a new building, but it is still the same repository, same collection, same institution.
The building Alsterterrasse 1 on the contrary is a Place, it does not move, but it hosts multiple repositories (there may be other libraries on other floors).
A manuscript may have been in AAI library and now in HLZ library - in this case it is NOT the repository that has moved but the manuscript has changed repositories.

Reply:
HLZ is an institution and repository. It is not a place, as it may change coordinates.
AT1 is a place. It is always here. Even if we are not.
In case of a church in Ethiopia, which does not move (a tabot may move but then to a new place/church, same for mss), you have a place and a repository with the same coordinates creating confusion for you. But a repository's coordinates are conceptually irrelevant/instable. A place's coordinates are always the same and should ideally always be provided.